src/components/Docs/SnippetViewer/WriteAuthzModelViewer.tsx

```diff
@@ -0,0 +1,70 @@
+import { getFilteredAllowedLangs, SupportedLanguage, LanguageMappings } from './SupportedLanguage';
+import { defaultOperationsViewer } from './DefaultTabbedViewer';
+import assertNever from 'assert-never/index';
+import { AuthorizationModel } from "@auth0/fga";
+
+
+interface WriteAuthzModelViewerOpts {
+  authorizationModel: AuthorizationModel;
+  skipSetup?: boolean;
+  allowedLanguages?: SupportedLanguage[];
+}
+
+function writeAuthZModelViewer(lang: SupportedLanguage, opts: WriteAuthzModelViewerOpts, languageMappings: LanguageMappings) {
+  switch (lang) {
+    case SupportedLanguage.CURL: {
+      return `curl -X POST $FGA_API_URL/stores/$FGA_STORE_ID/write \\
```

Comment:
Can we get these cases refactored out as testable pure functions? The `writeAuthZModelViewer` function is getting unruly.

Reply:
Refactored out